src/Octoshift/Services/ReclaimService.cs

```diff
@@ -0,0 +1,189 @@
+
+            var githubOrgId = await _githubApi.GetOrganizationId(githubTargetOrg);
+
+            var mannequins = await GetMannequins(githubOrgId);
+
+            foreach (var line in lines.Skip(1).Where(l => l != null && l.Trim().Length > 0))
+            {
+                var (login, userid, claimantLogin) = ParseLine(line);
+
+                if (login == null)
+                {
+                    continue;
+                }
+
+                if (!force && mannequins.IsClaimed(login, userid))
+                {
+                    _log.LogError($"{login} is already claimed. Skipping (use force if you want to reclaim)");
+                    continue;
+                }
+
+                var mannequin = mannequins.FindFirst(login, userid);
+
+                if (mannequin == null)
+                {
+                    _log.LogError($"Mannequin {login} not found. Skipping.");
+                    continue;
+                }
+
+                var claimantId = await _githubApi.GetUserId(claimantLogin);
+
+                if (claimantId == null)
+                {
+                    _log.LogError($"Claimant \"{claimantLogin}\" not found. Will ignore it.");
+                    continue;
+                }
+
+                var result = await _githubApi.ReclaimMannequin(githubOrgId, userid, claimantId);
+
+                HandleResult(login, claimantLogin, mannequin, claimantId, result);
+            }
+        }
+
+        private async Task<Mannequins> GetMannequins(string githubOrgId)
+        {
+            var returnedMannequins = await _githubApi.GetMannequins(githubOrgId);
+
+            return new Mannequins(returnedMannequins);
+        }
+
+        private void HandleResult(string mannequinUser, string targetUser, Mannequin mannequin, string targetUserId, MannequinReclaimResult result)
+        {
+            if (result.Errors != null)
+            {
+                _log.LogError($"Failed to reclaim {mannequinUser} ({mannequin.Id}) to {targetUser} ({targetUserId}) Reason: {result.Errors[0].Message}");
+                _failed = true;
+                return;
+            }
+
+            if (result.Data.CreateAttributionInvitation is null ||
+                result.Data.CreateAttributionInvitation.Source.Id != mannequin.Id ||
+                result.Data.CreateAttributionInvitation.Target.Id != targetUserId)
+            {
+                _log.LogError($"Failed to reclaim {mannequinUser} ({mannequin.Id}) to {targetUser} ({targetUserId})");
+                _failed = true;
+                return;
+            }
+
+            _log.LogInformation($"Successfully reclaimed {mannequinUser} ({mannequin.Id}) to {targetUser} ({targetUserId})");
+        }
+
+        private void HandleResult()
+        {
+            if (_failed)
+            {
+                throw new OctoshiftCliException("Failed to reclaim mannequin(s).");
+            }
+        }
+
+        private (string, string, string) ParseLine(string line)
```

Comment:
It'd be way more clear to name the fields:
```suggestion
        private (string Login, string UserId, string ClaimantLogin) ParseLine(string line)
```

Reply:
done, but used different naming to be more consistent with CSV header